arch/x86/crc32_vpclmulqdq.c

```diff
@@ -0,0 +1,17 @@
+/* crc32_vpclmulqdq.c -- VPCMULQDQ-based CRC32 folding implementation.
+ * Copyright Wangyang Guo (wangyang.guo@intel.com)
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#if defined(X86_PCLMULQDQ_CRC) && defined(X86_VPCLMULQDQ_CRC)
+
+#define X86_VPCLMULQDQ
+#define CRC32_FOLD_COPY_NAME  crc32_fold_vpclmulqdq_copy
```

Comment:
In the other template headers, I didn't put the word `_NAME` at the end of the macros... Do we need to here?

Reply:
The ``_NAME`` suffix has been removed.